src/translations/screens/subscreens/TravelToken.ts

```diff
@@ -495,7 +495,7 @@ export default orgSpecificTranslations(SelectTravelTokenTexts, {
             'Kan eg reise utan mobil?',
           ),
           answer: _(
-            'Du må alltid ha med deg det mobilen du har valgt å bruke billetten på.',
+            'Du må alltid ha med deg den mobilen du har valgt å bruke billetten på.',
```

Comment:
```suggestion
            'Du må alltid ha med deg den telefonen du har valgt å bruke billetten på.',
```

Reply:
Actually, this is an outside context fix on behalf of 'fram'. It's grammar correction and they should use 'mobilen' :-) If anything, this change should be left out.